pkgs/development/libraries/gssdp/default.nix

```diff
@@ -37,6 +39,7 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
+    gi-docgen
```

Comment:
This does not sound right. Maybe try moving pkg-config around like in 63793f516e5164eace848e37e4852a3d33e71f93.

Reply:
Hmm good idea, didn't consider moving `pkg-config` up in the sliding-window.

Reply:
okay I've added `pkg-config` to`depsBuildBuild`, and kept it in `nativeBuildInputs` as well since it's needed to find necessary libraries.